docs/source/usage/parameters.rst

```diff
@@ -236,6 +236,20 @@ Initial Beam Distributions
         * ``beam.muypy`` (``float``, dimensionless, default: ``0``) correlation Y-Py
         * ``beam.mutpt`` (``float``, dimensionless, default: ``0``) correlation T-Pt
 
+    * ``thermal`` for a 6D stationary thermal or bithermal distribution.
+      This distribution type is described, for example in:
+      R. D. Ryne et al, "A Test Suite of Space-Charge Problems for Code Benchmarking", in Proc. EPAC2004, Lucerne, Switzerland.
+      C. E. Mitchell et al, "ImpactX Modeling of Benchmark Tests for Space Charge Validation", in Proc. HB2023, Geneva, Switzerland.
+      With additional parameters:
+
+        * ``beam.k`` (``float``, in inverse meters) external focusing strength
+        * ``beam.kT`` (``float``, dimensionless) temperature of core population
+        * ``beam.kT_halo`` (``float``, dimensionless, default ``kT``) temperature of halo population
```

Comment:
@cemitch99 what is the temperature normalized to? The energy of the reference particle?

Reply:
Here kT = <(px/p0)^2> = <(py/p0)^2>, where p0 is the momentum of the reference particle.

Reply:
Shall we add this detail to the doc strings?